packages/cluster-ui/src/transactionDetails/transactionDetails.tsx

```diff
@@ -95,6 +109,95 @@ export class TransactionDetails extends React.Component<
                     arrayItemName={"statements for this transaction"}
                     activeFilters={0}
                   />
+                  <SummaryCard>
+                    <div className={summaryCardStylesCx("summary--card__item")}>
+                      <h3
```

Comment:
Here you could use the `<Heading />` component for `@cockroachlabs/ui-components`. It encompasses standard font treatments for the Design system. Usage could look something like, 

```jsx
import { Heading } from "@cockroachlabs/ui-components";

// .... 

<Heading type="h3" className={summaryCardStylesCx("summary--card__item--label")}>
  Mean transaction time
</h3>
```

Reply:
Thanks! Didn't realize I could see that in figma.